assets/settings/default.json

```
@@ -997,5 +997,16 @@
   //     ]
   //   }
   // ]
-  "ssh_connections": null
+  "ssh_connections": null,
+  // Configures the Context Server Protocol binaries
+  //
+  // Examples:
+  // {
+  //   "id": "server-1",
+  //   "executable": "/path",
+  //   "args": ['arg1", "args2"]
+  // }
+  "experimental.context_servers": {
```

```suggestion
  "experimental_context_servers": {
```

I like including "experimental in here" but I don't think we have a convention of using dots in key names - underscore seems fine to me! 😄

---

The dot is fine, it's what we do for `experimental.theme_overrides`.

---

TIL! 👍